languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,37 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rulegroup id="ARGOT_BEAUF_ET_BOF" name="argot beauf et bof" default="temp_off">
+          <rule>
+              <pattern case_sensitive="yes">
+                  <marker>
+                      <token regexp="yes">bof</token>
+                  </marker>
+              </pattern>
+              <message>Ce terme est une expression considérée comme familière. Souhaitez-vous la remplacer par le terme <suggestion>moyen</suggestion> employé dans le langage formel ?</message>
+              <example correction="moyen">C'est <marker>bof</marker> !</example>
+          </rule>
+          <rule>
```

Comment:
See rule editor results: An antipattern for `mon b(eau|o)f, ma s(œ|oe)ur et moi` seems to make sense (movie title).